deel-2-flexbox-layout/opdracht-5/styles/styles.css

```diff
@@ -1 +1,42 @@
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+main {
+    display: flex;
+    flex-basis: 900px;
+}
+
+header{
+    display: flex;
+    flex-basis: 180px;
+}
+
+article {
+    display: flex;
+    flex-basis: 1400px;
+    order: 2;
+}
+
+nav {
+    display: flex;
+    flex-basis: 500px;
+    order: 1;
+}
+
+aside {
+    display: flex;
+    flex-basis: 500px;
+    order: 3;
+}
+
+footer {
+    display: flex;
```

Comment:
regel 38: display: flex; zet je op meerdere plekken neer (bijv ok bij nav en aside) maar omdat ze direct binnen een flexcontainer vallen (namelijk de body of de main) zijn het al flex-items, je hoeft ze daarom geen display:flex meer mee te geven (is alleen maar overbodige codering :) )